code/svb-docker/python/svb_web/core/tests/test_account_model.py

```diff
@@ -0,0 +1,158 @@
+import pytest, time
+
+from core.models import Customer, Account, AnchorEvent
+from datetime import datetime, timedelta, timezone
+
+@pytest.mark.django_db
+def test_account_create_delete():
+    customer = Customer(
+        customer_id="test_cust",
+        first_name="frank",
+        costume="lyboring",
+        referrer=None,
+        security_candy="toots"
+    )
+    account1 = Account(
+        account_name="test1",
+        customer=None
+    )
+    account2 = Account(
+        account_name="test2",
+        customer=None
+    )
+    account1.save()
+    account2.save()
+
+    # Test that account numbers are incrementing properly.
+    assert account1.account_number < account2.account_number
+    # Test that accounts can be retrieved by name.
+    assert Account.objects.filter(account_name__exact="test1")[0] == account1
+    # Test that accounts can be deleted.
+    account1.delete()
+    assert len(Account.objects.filter(account_name__exact="test1")) == 0
+    
+    # Reset database to original state.
+    account2.delete()
+
+@pytest.mark.django_db
+def test_account_init_delete():
+    account1 = Account(
+        account_name="test1",
+        customer=None
+    )
+    account1.save()
+    # Account not yet initialized, there should be no associated anchor events.
+    assert len(AnchorEvent.objects.filter(account__exact=account1)) == 0
+    
+    # Test that account can't be initialized in the future.

```

Comment:
best to separate these into multiple tests

Reply:
good feedback but no time gogogo